game/templates/game/landing.html

```
@@ -1,5 +1,14 @@
 {% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)